app/scripts/metamask-controller.js

```diff
@@ -95,6 +95,7 @@ module.exports = class MetamaskController extends EventEmitter {
       initState: initState.KeyringController,
       ethStore: this.ethStore,
       getNetwork: this.networkController.getNetworkState.bind(this.networkController),
+      encryptor: opts.platform.encryptor || undefined,
```

Comment:
the platform obj is intended to perform the same operation on different platforms

the encryptor should be configurable independent platform (e.g. cryptoX in electron, cryptoY in electron)

Reply:
That's why I've made it configurable here. Is there a problem you see with this implementation?

Reply:
In the docs it describes the required API for the encryptor object.

Reply:
the encryptor should not be inside the platform obj